e2e-tests/mongod-major-upgrade/conf/operator.main.psmdb-operator.json

```diff
@@ -0,0 +1,36 @@
+{
+  "versions": [
+    {
+      "operator": "1.14.0",
+      "product": "psmdb-operator",
+      "matrix": {
+        "mongod": {
+        },
+        "pmm": {
+          "2.27.0": {
+            "image_path": "percona/pmm-client:2.27.0",
+            "image_hash": "4f4d4508afe4ef3b5d72e2ebec6485be84204902b0b2b23c3a9e7c2fe4726884",
+            "status": "recommended",
+            "critical": false
```

Comment:
Why 2.27?

Reply:
It's just boilerplate version, pmm is not used in this test.